whelk-core/src/main/groovy/whelk/component/PostgreSQLComponent.groovy

```diff
@@ -336,18 +335,6 @@ class PostgreSQLComponent implements Storage {
 
         GET_MAX_MODIFIED = "SELECT MAX(modified) from $mainTableName WHERE id IN (?)"
```

Comment:
Just a thought popping up when reading all these min-max-dep-modified statements: wouldn't it be nice to call it latest modified instead? For some reason my brain always does: max modified = maximally modified, oh wait...

Reply:
Ditto

Reply:
In my brain, the column is a numeric value (sort of), and we constantly use < > (more than, less than) operators on them. And so, max/min makes the most sense to me. But, im not religious on the matter.